src/OmniSharp.Script/FileParserResult.cs

```diff
@@ -0,0 +1,20 @@
+using System.Collections.Generic;
+
+namespace OmniSharp.Script
+{
+    public class FileParserResult
+    {
+        public FileParserResult()
+        {
+            Namespaces = new HashSet<string>();
+            References = new HashSet<string>();
+            LoadedScripts = new HashSet<string>();
+        }
+
+        public HashSet<string> Namespaces { get; }
```

Comment:
FYI that read-only auto-props can have initializers. So you could just have this with no constructor:

```C#
public HashSet<string> Namespaces { get; } = new HashSet<string>();
public HashSet<string> References { get; } = new HashSet<string>();
public HashSet<string> LoadedScripts { get; } = new HashSet<string>();
```

Your choice.